packages/server/graphql/mutations/archiveOrganization.ts

```diff
@@ -1,5 +1,6 @@
 import {GraphQLID, GraphQLNonNull} from 'graphql'
 import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import {removeTeamLimitsJobs} from '../../billing/helpers/teamLimitsCheck'
```

Comment:
+1 I think this indicates that `teamLimitsCheck` should be in its own file

Reply:
@Dschoordsch do you mean move ‘removeTeamLimitsJobs’ in a separate file?

Reply:
Yes, that's what I meant and almost what I wrote.